### PR TITLE
Updated

### PR DESCRIPTION
**Pull request by GABRYCA**
Updated, if some mistakes are found, they'll be fixed soon as possible

### CODE REVIEW

File: Translations/Base_Italian.lang [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
 #GUI_SETTINGS_MYSQL_SETUP_TITLE_FIELDNAME: MySQL - %1
-#GUI_PAGEABLE_PREVIOUS_ACTION: Click to go to the previous page
-#GUI_SETTINGS_MYSQL_SAVE_ACTION: Click to **test** and **save**
-#GUI_SETTINGS_MYSQL_NAME: MySQL Database
-#GUI_SETTINGS_MYSQL_SETUP_ACTION: Click to enable MySQL
-#GUI_SETTINGS_MYSQL_SETUP_TITLE_TYPEINVALUE: Type in the value
-#GUI_SETTINGS_MYSQL_SETUP_TITLE: Setup MySQL
+#GUI_PAGEABLE_PREVIOUS_ACTION: Clicca per andare alla pagina precedente
+#GUI_SETTINGS_MYSQL_SAVE_ACTION: Clicca per **test** e **save**
+#GUI_SETTINGS_MYSQL_NAME: MySQL **Database**
```
Comment (rlnt) [inline Translations/Base_Italian.lang]: Please keep the format of the original phrases. That means no addition of ** in your language. Please remove them

File: Translations/Base_Italian.lang [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
+#GUI_SETTINGS_MYSQL_SETUP_ACTION: Clicca per abilitare MySQL
+#GUI_SETTINGS_MYSQL_SETUP_TITLE_TYPEINVALUE: Digita il valore
+#GUI_SETTINGS_MYSQL_SETUP_TITLE: **Setup** MySQL
```
Comment (rlnt) [inline Translations/Base_Italian.lang]: Remove the ** here as well since they are not included in the original phrases

**Review by rlnt — CHANGES_REQUESTED** [review]
Please remove the * symbols in your phrases so it is like the original language.

**Comment by GABRYCA** [comment]
Should be fixed, the Highlight it's making me confusion

**Comment by rlnt** [comment]
The things I mentioned are fixed but you doubled some lines now. 14-34 and 57-66 are doubled

**Comment by GABRYCA** [comment]
It's because the given file to edit was "fuse" with an older commit

**Comment by timokueck** [comment]
To me it looks like the Base_Italian file is already fully translated. Ill close this now

**Pull request closed without merging**